modules/apps/dispatch/dispatch-lang/src/main/resources/content/Language.properties

```
@@ -24,6 +24,7 @@ runtime=Runtime
 scheduled-jobs=Scheduled Jobs
 single-node=Single Node
 task-execution-cluster-mode=Task Execution Cluster Mode
+task-executor-is-not-ready=Task executor is not ready
```

Missing period here.

```suggestion
task-executor-is-not-ready=Task executor is not ready.
```